File: AtCoder/第一回アルゴリズム実技検定A_-_2_倍チェック.py
result = 0
try:
  for i,s in enumerate(input()):
    n = int(s)
    if i == 0:
      n *= 100
      n *= 2
    elif i == 1:
      n *= 10
      n *= 2
    else:
      n *= 2
    result += n
  print(result)
except:
    print("error")

File: AtCoder/contest/第一回アルゴリズム実技検定A_-_2_倍チェック.ts
import * as fs from 'fs';

const input: string = fs.readFileSync('/dev/stdin', 'utf8').trim();

try {
  let result = 0;
  const twiceVal = input.split('').reduce((acc, curr, idx) => {
    if (isNaN(Number(curr))) {
      throw new Error('not number');
    }

    if (idx === 0) {
      return acc + Number(curr) * 200;
    } else if (idx === 1) {
      return acc + Number(curr) * 20;
    } else {
      return acc + Number(curr) * 2;
    }
  }, 0);
  console.log(twiceVal.toString());
} catch (err) {
  console.log('error');
}
